src/components/AddCardComponent/index.view.jsx

```diff
@@ -0,0 +1,38 @@
+import React , {useState, useEffect} from 'react';
+import AddCardComponentData from './index.data';
+
+//this one should be in data file when possible
+import db from '../../db/dexie'
+
+const AddCardComponent = () => {
+    //set the state and property
+    const [cardTitle, setCardTitle] = useState("");
+    const [cardContent, setCardContent] = useState("");
+	
+    //this function should be in another file. Keeping here temporarily 
+    const getPostInfo = (e) => {
```

Comment:
This function is not used in this file, right? Have you made sure that this function works though?

Reply:
I am using this for the onSubmit for the form. I want to shift this to the index.data.jsx but I'm not sure how to do that yet.